stackanetes/manifest.py

```diff
@@ -37,24 +37,30 @@ def __init__(self, configuration, service_dir):
         self.host_interface = CONF.stackanetes.minion_interface_name
         self.external_ip = CONF.stackanetes.external_ip
         self.image_prefix = CONF.stackanetes.image_prefix
+        self.host_network = configuration.get('host_network', True)
         self.namespace = CONF.stackanetes.namespace
+        self.ports = configuration.get('ports', [])
+        for port in self.ports:
+            if not port.get('target_port'):
+                port['target_port'] = port.get('port')
         if configuration.get('containers'):
             self._parameters_for_multi_containers_pod(configuration)
         else:
-            self._parameters_for_singel_container_pod(configuration)
+            self._parameters_for_single_container_pod(configuration)
         self.template_name = self._find_template()
 
-    def _parameters_for_singel_container_pod(self, configuration):
-        self.command = configuration['command']
-        self.configmaps = configuration['files']
+    def _parameters_for_single_container_pod(self, configuration):
+        self.command = configuration.get('command')
+        self.configmaps = configuration.get('files', [])
         self.image = configuration.get('image')
-        self.ports = configuration.get('ports', [])
         self.envs = configuration.get('envs', [])
+        self.privileged = configuration.get('privileged', True)
         self.session_affinity = configuration.get("session_affinity",[])
         self.non_root = configuration.get("non_root",[])
         self.emptydirs = configuration.get('emptyDirs', [])
         self.mounts = configuration.get('mounts', [])
-        self.envs.append({'COMMAND': self.command})
```

Comment:
Remove that changes. Entrypoint requires that env so it must be set. But I agree that we need to add some service_validation method.